service/src/test/java/tds/config/web/endpoints/HealthControllerTest.java

```diff
@@ -0,0 +1,29 @@
+package tds.config.web.endpoints;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HealthControllerTest {
```

Comment:
Yeah, I just noticed it was there and added it because I can't help myself adding test coverage.